languages/ruby/lib/oso/polar/polar.rb

```diff
@@ -181,8 +236,8 @@ def query_rule(name, *args)
       # under a previously-registered name.
       # @raise [FFI::Error] if the FFI call returns an error.
       # @return [self] for chaining.
-      def register_class(cls, name: nil)
-        name = host.cache_class(cls, name: name || cls.name)
+      def register_class(cls, name: nil, fields: {}, fetcher: nil)
```

Comment:
This is going to have to change a bit with the enforcer stuff, but I suppose that's a job for next cycle